source/lac/petsc_parallel_vector.cc

```diff
@@ -85,6 +85,15 @@ namespace PETScWrappers
 
 
 
+    Vector::Vector(const Vector &v)
+      : VectorBase(v)
+      , communicator(v.communicator)
```

Comment:
I would skip these and just leave it completely up to `operator=`.

Reply:
Let me keep the former because the compiler wants me to call the base constructor (but I choose the empty one) and remove the latter.

Reply:
ah. Lets keep it then.